Header header
float64[] deltas
float64[] delta_errors
float64 fft_sharpness
float64 heading
float64 declination
